invenio_records_resources/errors.py

```diff
@@ -10,10 +10,40 @@
 
 import json
 
+from flask import abort, current_app, jsonify, make_response, request, url_for
 from invenio_rest.errors import RESTException, RESTValidationError
+from werkzeug.routing import BuildError
 
 # TODO: Revise this
 
+
+def create_pidredirected_error_handler(route):
```

Comment:
moderate:  below is called `pid_redirected`, consistency wise I would say both with _ or without (my take is with)

```suggestion
def create_pid_redirected_error_handler(route):
```